src/components/block/index.ts

```diff
@@ -642,22 +644,36 @@ export default class Block extends EventsDispatcher<BlockEvents> {
   }
 
   /**
-   * Enumerates initialized tunes and returns fragment that can be appended to the toolbars area
-   *
-   * @returns {DocumentFragment[]}
+   * Returns data to render in tunes menu.
+   * Splits block tunes settings into 2 groups: popover items and custom html.
    */
-  public renderTunes(): [DocumentFragment, DocumentFragment] {
-    const tunesElement = document.createDocumentFragment();
-    const defaultTunesElement = document.createDocumentFragment();
+  public getTunes(): [TunesMenuEntry[], HTMLElement] {
+    const tunesElement = document.createElement('div');
+    let tunesItems: PopoverItem[] = [];
 
-    this.tunesInstances.forEach((tune) => {
-      $.append(tunesElement, tune.render());
-    });
-    this.defaultTunesInstances.forEach((tune) => {
-      $.append(defaultTunesElement, tune.render());
+    const defaultTunesInstances = Array.from(this.defaultTunesInstances.values());
+    const customTunesInstances = Array.from(this.tunesInstances.values());
+    const tunesDefinedInTool =
+      typeof this.toolInstance.renderSettings === 'function'
+        ? [ this.toolInstance.renderSettings() ].flat()
+        : [];
+    const otherTunes = customTunesInstances.concat(defaultTunesInstances).map(item => item.render());
```

Comment:
naming is not correct since there is a combination of custom tunes and default tunes as I can see.